website/docs/getting-started.md

```diff
@@ -262,7 +254,7 @@ of strings. For example, `pd.Series([["test"], ["token2", "token3"]])` is a vali
 3. __VectorSeries__: Every cell is a vector representing text, i.e.
 a list of floats. For example, `pd.Series([[1.0, 2.0], [3.0]])` is a valid VectorSeries. Most dimensionality reduction functions, like `pca` will take VectorSeries as an input and also return a VectorSeries.
 
-4. **DataFrame**: A Pandas DataFrame where the rows can be the documents and the columns can be the words/terms in all the documents. 
+We also return a **DataFrame**, which basically is a Pandas DataFrame. Those DataFrames are used to store some relations. For example `count` will return a DataFrame where the rows are the documents and the columns are be the words/terms in all the documents. 
```

Comment:
Some functions also return a Pandas **DataFrame**. The DataFrame then represents a matrix. For example `count` ...

Reply:
adaped to the tutorial 🍖